<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 237, 231);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseFileBut">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Levenim MT</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(38, 0, 61);
font: 75 11pt &quot;Levenim MT&quot;;
background-color: rgb(234, 245, 237);
border-radius: 1px;
border: 1px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 1px solid #26003d;
</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printBut">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>325</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Levenim MT</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(38, 0, 61);
font: 75 12pt &quot;Levenim MT&quot;;
background-color: rgb(234, 245, 237);
border-radius: 1px;
border: 1px solid #26003d;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)}
QPushButton:hover {background-color: rgba(200, 229, 225, 0.7); 
border: 1px solid #26003d;
</string>
    </property>
    <property name="text">
     <string>Распечатать</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Levenim MT</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Levenim MT&quot;;</string>
    </property>
    <property name="text">
     <string>Выберите файл конфигурации формата *.txt</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 10pt &quot;Levenim MT&quot;;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
